<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>smallwindow</class>
 <widget class="QWidget" name="smallwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>277</width>
    <height>156</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>4</y>
     <width>271</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget {
background-color: rgba(194, 194, 194, 0.97);
border-radius: 28px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>198</x>
     <y>7</y>
     <width>71</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border:none;
}
QPushButton:hover{
	padding-bottom:4px;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="res/ui1.qrc">
        <normaloff>:/icons/hide.png</normaloff>:/icons/hide.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_4">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="res/ui1.qrc">
        <normaloff>:/icons/exit.png</normaloff>:/icons/exit.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>83</x>
     <y>24</y>
     <width>111</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton{
	border:none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="res/ui1.qrc">
     <normaloff>:/icons/picker.ico</normaloff>:/icons/picker.ico</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>64</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_3{
    color: rgb(98, 105, 112);
	font-size: 13px; 
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>23</y>
     <width>231</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">* {
    color: rgb(98, 105, 112);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>pushButton_7</zorder>
  <zorder>label_3</zorder>
  <zorder>label_2</zorder>
  <zorder>frame</zorder>
 </widget>
 <resources>
  <include location="res/ui1.qrc"/>
 </resources>
 <connections/>
</ui>
